bindings/rust/bench/historical-perf/bench-past.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# suppress stdout
+exec >/dev/null
+export CARGO_TERM_QUIET=true
+export RUSTFLAGS=-Awarnings
```

Comment:
can you add a comment why you need these

Reply:
I'll add that we're suppressing cargo warnings to the suppress stdout comment.